site/guide/organize-model-inventory.qmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Organize models in the inventory"
+---
+
+Customize the model inventory to fit your needs.
+
+## Prerequisites
+
+Any user can complete these steps.
+
+## Steps
+
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
+
+2. Navigate to the **Model Inventory**.
+
+3. Use the top buttons to organize the model inventory to your preference:
```

Comment:
Check if the icons are available via [Font Awesome](https://fontawesome.com/) and add them via `{{< fa ICON_NAME >}}`? If not all of them, maybe you could ask what the icon library is, we might be able to add it to Quarto.

Reply:
I added the icons, but the card view icon needs premium - unless I can find it in another library for free >:) Which channel should I ping about the icon library?

Reply:
Some of the icons in the UI are Figma iconsets, which I'm not certain we can easily insert unless we cap the icon and upload it as an image, which could get messy real quick.